core/tests/performance_tests/base.py

```diff
@@ -119,3 +143,79 @@ def _test_page_load_time_for_cached_session(self):
         self.assertLessEqual(
             self.page_metrics.get_average_page_load_time_millisecs(),
             self.load_time_limit_cached_ms)
+
+    def _get_driver(self):
+        return self.data_fetcher.setup_driver()
+
+    def _check_login(self, driver):
+        driver.get(self.BASE_URL)
+        time.sleep(5)
+        resulting_url = driver.current_url
+
+        if resulting_url == self.BASE_URL + self.SPLASH_URL_SUFFIX:
```

Comment:
I feel like it's more pythonic to use interpolation instead of concatenation. Ditto elsewhere.